topics/transcriptomics/tutorials/scrna-case_monocle3-trajectories/slides.html

```diff
@@ -0,0 +1,463 @@
+---
+layout: tutorial_slides
+logo: "GTN"
+
+title: "Trajectory analysis"
+questions:
+  - "What is trajectory analysis?"
+  - "What are the main methods of trajectory inference?"
+  - "How are the decisions about the trajectory analysis made?"
+  - "What to take into account when choosing the method for your data?" 
+objectives:
+  - "Become familiar with the methods of trajectory inference"
+  - "Learn how the algorithms produce outputs"
+  - "Be able to choose the method appropriate for your specific data"
+  - "Gain insight into methods currently available in Galaxy"
+
+subtopic: writing
+key_points:
+  - "Trajectory analysis in pseudotime is a powerful way to get insight into the differentiation and development of cells."
+  - "There are multiple methods and algorithms used in trajectory analysis and depending on the dataset, some might work better than others."
+  - "Trajectory analysis is quite sensitive and thus you should always check if the output makes biological sense."
+contributors:
+  - wee-snufkin
+requirements:
+  -
+    type: "internal"
+    topic_name: transcriptomics
+    tutorials:
+        - scrna-intro
+
+---
+
+### What is trajectory analysis?
+.footnote[[Deconinck et al. Recent advances in trajectory inference from single-cell omics data](https://doi.org/10.1016/j.coisb.2021.05.005.)]
```

Comment:
It may be out of scope (and feel free to reject if so!) but if you'd like to add speaker notes, for what _you_'d say about each slide if you were presenting them @wee-snufkin, then we can automatically generate a video from that. 

[How it works](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/slides-with-video/tutorial.html), and an [example of it being used](https://training.galaxyproject.org/videos/watch.html?v=transcriptomics/tutorials/scrna-plates-batches-barcodes/slides) / [example of the source](https://github.com/galaxyproject/training-material/blob/main/topics/transcriptomics/tutorials/scrna-plates-batches-barcodes/slides.html#L66-L70)

Reply:
That's a good shout - I've added some presenter notes, unfortunately not to all the slides - and for now they are more like actual notes rather than a transcript. I'm afraid I won't be able to do it quite soon, but maybe at some point I'll come back to that!